iwf/object_encoder.py

```diff
@@ -291,7 +291,7 @@ def default(self, o: Any) -> Any:
         See :py:meth:`json.JSONEncoder.default`.
         """
         # Dataclass support
-        if dataclasses.is_dataclass(o):
+        if dataclasses.is_dataclass(o) and not isinstance(o, type):
```

Comment:
Fixes mypy error

Reply:
I didn't see this error before. 
Did you find this fix somewhere? can you put a reference here for more context?